bitnami/grafana/templates/deployment.yaml

```diff
@@ -79,7 +79,12 @@ spec:
           image: {{ include "volumePermissions.image" . }}
           imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
           command:
-            - chown -R {{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }} /bitnami/grafana
+            - /bin/bash
+            - '-ec'
+          args:
+            - |
+              mkdir -p /bitnami/grafana
```

Comment:
It should be mounted, but without this change we get this error:
```
failed to create shim: OCI runtime create failed:
container_linux.go:380: starting container process caused: exec:
"chown -R 1001:1001 /bitnami/grafana": stat chown -R 1001:1001
/bitnami/grafana: no such file or directory: unknown
```

I just experimented some more, and it also works without `mkdir -p`, however for some reason it does not work directly as a `command`, it needs to be executed in bash to avoid the error. I can't think of any reason for that.